### PR TITLE
Nodes: fix getNetworkId() and types

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Fix getNetworkId(), if parsed as number it can be wrong because it's a u64
Also fixed
- `Transaction::networkId` type;
- `FilterOptions::outputTypes` type;

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Running an example with client.getNetworkId() which returned a wrong value before

### CODE REVIEW

File: bindings/wasm/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.7 - 2023-09-14
```
Comment (thibault-martinez) [inline bindings/wasm/CHANGELOG.md]: 1.0.6 is still unreleased

Comment (Thoralf-M) [reply bindings/wasm/CHANGELOG.md]: Oh right

File: bindings/nodejs/CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.11 - 2023-09-14
+
+### Fixed
+
+- `Client::getNetworkId()`;
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
- `Client::getNetworkId()` return type;
```

File: bindings/nodejs/CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `Transaction::networkId` type;
```
Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
- `RegularTransactionEssence::networkId` type;
```

File: bindings/nodejs/lib/types/wallet/transaction-options.ts
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/wallet/transaction-options.ts:1]: Changelog?

Comment (Thoralf-M) [reply bindings/nodejs/lib/types/wallet/transaction-options.ts:1]: Don't think that's needed, they're still all a string, just another type alias

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]

**Pull request merged**